lib/stacker/stack/parameter.rb

```diff
@@ -63,6 +63,10 @@ def to_s
         end
       end
 
+      def ==(parameter)
+        parameter.value == value && parameter.region == region
```

Comment:
You might also check that the argument is the same class, but maybe that's too defensive.